src/translations/locales/en/auth.json

```
@@ -64,7 +64,8 @@
       "default": "Invalid address"
     },
     "tooManyAttemptsTryAgain": "Too many attempts, please try again in {{waitDuration}} seconds.",
-    "missingData": "Please specify the {{missingData}}"
+    "missingData": "Please specify the {{missingData}}",
+    "registrationApiFailedWithNoReason": "Registration API failed with no reason"
```

Is this something that is displayed to the user? If it is ... we might need to make it a little bit less technical. Maybe:

```suggestion
    "registrationApiFailedWithNoReason": "An unexpected problem has occurred - please contact our support team"
```

---

👍